src/components/modal/index.tsx

```diff
@@ -79,12 +78,7 @@ class Modal extends Component<ModalProps> {
   }
 
   renderTouchableOverlay() {
-    const {
-      testID,
-      overlayBackgroundColor = Colors.$backgroundDefault,
-      onBackgroundPress,
-      accessibilityLabel = 'Dismiss'
-    } = this.props;
+    const {testID, overlayBackgroundColor, onBackgroundPress, accessibilityLabel = 'Dismiss'} = this.props;
```

Comment:
This change doesn't seem to be related to your PR, maybe you should merge master?